src/index.css

```diff
@@ -1,68 +1,65 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
 :root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
-  line-height: 1.5;
-  font-weight: 400;
-
-  color-scheme: light dark;
-  color: rgba(255, 255, 255, 0.87);
-  background-color: #242424;
-
-  font-synthesis: none;
-  text-rendering: optimizeLegibility;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  --background-color: #ffffff; /* white */
```

Comment:
This is good, but we could've used the MUI theme to handle the light/dark mode instead. We can also choose closer colors from the existing MUI theme and the dark/light mode will be handled by them and we won't need to handle it ourselves, but this will require that we use the colors in our component from theme.

Reply:
i add theme with mui but not finish yet